python-numpy-basics/Sets and Tuples.ipynb

```diff
@@ -0,0 +1,570 @@
+{
```

Comment:
<p>typo error 'mod ified', correct it</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SRA-VJTI/Intro_to_CV/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>